packages/langium/src/parser/langium-parser.ts

```diff
@@ -302,8 +305,12 @@ class ChevrotainWrapper extends EmbeddedActionsParser {
 
     private analysed = false;
 
-    constructor(tokens: TokenType[]) {
-        super(tokens, { recoveryEnabled: true, nodeLocationTracking: 'onlyOffset' });
+    constructor(tokens: TokenType[], config?: IParserConfig) {
+        super(tokens, {
+            recoveryEnabled: true,
+            nodeLocationTracking: 'onlyOffset',
+            ...config
```

Comment:
I'm wondering about this syntax being allowed although `config` is potentially undefined. I couldn't find any specification about that.

Reply:
I've tried it a playground -- it's working nicely.